pandas/_libs/join.pyx

```diff
@@ -268,7 +268,7 @@ ctypedef fused join_t:
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def left_join_indexer_unique(join_t[:] left, join_t[:] right):
+def left_join_indexer_unique(ndarray[join_t] left, ndarray[join_t] right):
```

Comment:
maybe these need const?

Reply:
we can't use `const join_t` until cython3